src/Motives/MotiveButtons.js

```diff
@@ -1,33 +1,149 @@
 import React from 'react'
 import styled from 'styled-components'
-import Button from '../common/Button'
+import InfoOverlay from '../common/InfoOverlay'
 
 export default function MotiveButtons({ handleMotiveClick }) {
-  const motives = [
-    'Anerkennung',
-    'Wichtigkeit',
-    'Verlässlichkeit',
-    'Solidarität',
-    'Autonomie',
-    'Grenzen',
+  const motiveData = [
+    {
+      name: 'Anerkennung',
+      description: () => {
```

Comment:
Interesting that you are storing fixed information in an object using an arrow function. Why are you doing that?  Will there be functionality added later on?

Reply:
Nice short pull request. I found it semantically very clear to read.

Reply:
> Interesting that you are storing fixed information in an object using an arrow function. Why are you doing that? Will there be functionality added later on?

I did that because  you can't store JSX in a string. I will probably create a separate component for the overlay contents later but at the moment I'm still waiting for the final text.